app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+        self,
+        name: str,
+        health: int = 100,
+        hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        result = f"Name: {self.name}, \
+Health: {self.health}, \
+Hidden: {self.hidden}"
+        return "{" + result + "}"
```

Comment:
Remake this, you can use brackets and remove `\`. And you can use `{{}}` in your f-string to display it like { }

Reply:
what brackets???

Reply:
I don't understand how to do that